docs/api/hooks.md

```diff
@@ -68,16 +67,71 @@ export const CounterComponent = () => {
 Using props to determine what to extract:
 
 ```jsx
-import React from 'react'
+import React, {useCallback} from 'react'
 import { useSelector } from 'react-redux'
 
-export const TodoListItem = props => (
-  const todo = useSelector(state => state.todos[props.id], [props.id])
+export const TodoListItem = ({id}) => (
+  const selector = useCallback(state => state.todos[id], [id])
+  const todo = useSelector(selector)
```

Comment:
for this example it makes no sense to `useCallback` and it makes the example more complicated than it needs to be. Just do `useSelector(state => state.todos[id])` and leave the whole memoization stuff for other examples